dev/ci/ci-basic-overlay.sh

```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
 ########################################################################
 project struct_tact "https://github.com/uwplse/StructTact" "master"
+# Contact @palmskog on github
 
 project inf_seq_ext "https://github.com/DistributedComponents/InfSeqExt" "master"
+# Contact @palmskog on github
 
 project cheerios "https://github.com/uwplse/cheerios" "master"
+# Contact @palmskog on github
 
 project verdi "https://github.com/uwplse/verdi" "master"
+# Contact @palmskog on github
 
 project verdi_raft "https://github.com/uwplse/verdi-raft" "master"
+# Contact @palmskog on github
 
 ########################################################################
 # stdlib2
 ########################################################################
 project stdlib2 "https://github.com/coq/stdlib2" "master"
+# Contact @maximedenes, @vbgl on github
 
 ########################################################################
 # argosy
 ########################################################################
 project argosy "https://github.com/mit-pdos/argosy" "master"
+# Contact @tchajed on github
 
 ########################################################################
 # ATBR
 ########################################################################
 project atbr "https://github.com/coq-community/atbr" "master"
+# Contact @palmskog, @tchajed on github
 
 ########################################################################
 # perennial
 ########################################################################
 project perennial "https://github.com/mit-pdos/perennial" "coq/tested"
+# Contact @upamanyus, @RalfJung on github
 
 ########################################################################
 # metacoq
 ########################################################################
 project metacoq "https://github.com/MetaCoq/metacoq" "main"
+# Contact @mattam82, @yforster on github
 
 ########################################################################
 # SF suite
 ########################################################################
 project sf "https://github.com/DeepSpec/sf" "master"
+# Contact @bcpierce00 on github
 
 ########################################################################
 # Coqtail
 ########################################################################
 project coqtail "https://github.com/whonore/Coqtail" "master"
+# Contact @whonore on github
 
 ########################################################################
 # Deriving
 ########################################################################
 project deriving "https://github.com/arthuraa/deriving" "master"
+# Contact @arthuraa on github
 
 ########################################################################
 # VsCoq
 ########################################################################
 project vscoq "https://github.com/coq-community/vscoq" "coq-master"
+# Contact @maximedenes, @huynhtrankhanh, @thery, @Blaisorblade on github
```

Comment:
```suggestion
# Contact @rtetley on github
```
I think for this part of VsCoq it's only @rtetley these days, maybe @gares as well?